tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/handler/__bindingIdCamelCase__Handler.java

```diff
@@ -20,28 +23,28 @@
 /**
  * The {@link ${bindingIdCamelCase}Handler} is responsible for handling commands, which are
  * sent to one of the channels.
- * 
+ *
  * @author ${author} - Initial contribution
  */
 public class ${bindingIdCamelCase}Handler extends BaseThingHandler {
 
-    private Logger logger = LoggerFactory.getLogger(${bindingIdCamelCase}Handler.class);
+    private final Logger logger = LoggerFactory.getLogger(${bindingIdCamelCase}Handler.class);
```

Comment:
@martinvw Thanks for the links but I assume I know that stuff already :wink: 
If written "we could think about", not that you should change it.

> imho it should never be used

Hm, currently I use it if I have an abstract base class that is used by a lot of sub class and I want to enable the logging of code that resides in the base class just for one sub class.
At the moment I don't see an anti-pattern or "never use it" but perhaps a "normally not needed".

If you have an idea how to implement the my example given above "better", feel free to send me a mail (I will be interested in). I assume there is no need to discuss that "off-topic" here.